bda602_hw1/hw1.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+import plotly
+import numpy as np
+import sklearn
+import plotly.express as px
+
+
+#r"/mnt/d/Data/CSV_file.csv"
+datafile ="http://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data"
+#c=pd.read_csv(datafile)
+#datafile2 = "/mnt/C:/Users/thoma/OneDrive/Documents/bda602/hw1/bezdekIris.data"
+#print(datafile)

```

Comment:
This prints the URL not the dataframe, it should be print(c). anyway it's not important just a note